examples/fluent-bit/amazon-opensearch-serverless/README.md

```diff
@@ -0,0 +1,21 @@
+### FireLens Example: Logging to Amazon OpenSearch Serverless Service with Fluent Bit
+
+Amazon OpenSearch Serverless is new offering that eliminates your need to manage OpenSearch instances. All existing Fluent Bit OpenSearch output plugin options work with OpenSearch Serverless. The only difference with serverless from a Fluent Bit POV is that you must specify the service name as aoss (Amazon OpenSearch Serverless) when you enable AWS_Auth:
+
+```
+AWS_Auth On
+AWS_Region <aws-region>
+AWS_Service_Name aoss
+```
+
+When sending logs to OpenSearch Serverless, your task role (e.g. `arn:aws:iam::XXXXXXXXXXXX:role/ecs_task_iam_role`) needs [OpenSearch Serverless Data Access permisions](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-data-access.html), not IAM policies. No IAM policies need to be attached to your task role. Give your task role the following Data Access permissions to your serverless collection:
+
+```
+aoss:CreateIndex
+aoss:UpdateIndex
+aoss:WriteDocument
+```
+
+See the [Opensearch Serverless Documentation on Data Access Policies](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/serverless-clients.html#serverless-ingestion-permissions).
```

Comment:
The link shoud say See Data access control for Amazon OpenSearch Serverless in the *OpenSearch Service Developer Guide*.

Reply:
Moved to the bottom of the document, but updated the text.